### PR TITLE
Update Brave cookies settings and move social media blocking

### PR DESCRIPTION
**Pull request by redoomed1**
Changes proposed in this PR:

- Update the Recommended Configuration section under Brave to reflect Brave's updated settings for sanitizing and storing cookies, as noted here: https://discuss.privacyguides.net/t/changes-in-brave-cookie-settings-and-new-script-blocking-feature/16590
- Move the Social media blocking section to Shields as it is no longer resides in its own section on the sidebar in brave://settings

(I originally submitted a PR on [Codeberg](https://codeberg.org/privacyguides/privacyguides.org/pulls), but it has returned Internal Server Errors for the past week or so.)

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

**Review by rollsicecream — APPROVED** [review]
LGTM

File: docs/desktop-browsers.md [hunk 1]
```diff
@@ -291,7 +291,6 @@ Brave allows you to select additional content filters within the internal `brave
 1. This option provides functionality similar to uBlock Origin's advanced [blocking modes](https://github.com/gorhill/uBlock/wiki/Blocking-mode) or the [NoScript](https://noscript.net/) extension.
 2. If you wish to stay logged in to a particular site you visit often, you can set exceptions on a per-site basis by clicking on the Shield icon in the address bar.
 
-##### Social media blocking
 
 - [ ] Uncheck all social media components
```
Comment (redoomed1) [inline docs/desktop-browsers.md:295]: Did I make a formatting error here? The Deploy Preview does not show "Uncheck all social media components" on this page at all.

Comment (rollsicecream) [reply docs/desktop-browsers.md:295]: Maybe don't put any space before `- [ ] Uncheck all social media components`. I'm not sure though but try that.

Comment (redoomed1) [reply docs/desktop-browsers.md]: Thanks for your feedback, @rollsicecream! I always appreciate another pair of eyes on anything written.

I looked at the current [code for the Brave section of the Mobile Browsers page](https://github.com/privacyguides/privacyguides.org/blob/main/docs/mobile-browsers.md?plain=1#L99) as a template for the change I want to submit.

**Comment by rollsicecream** [comment]
It does seem to work now, is it what you wanted?

**Comment by redoomed1** [comment]
> is it what you wanted?

Yes.

**Review by rollsicecream — APPROVED** [review]
LGTM so I re-approve.

**Review by dngray — APPROVED** [review]
Can confirm this has moved

**Pull request merged**